db.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
Do we not need an ID on startYears as well? 
And if we know some entities that we need sample data for testing maybe we could add them like:
- Courses
- Modules
- etc?

Reply:
I think we can add those data when we know the APIs